app/controllers/pages_controller.rb

```diff
@@ -27,6 +27,7 @@ def index
   end
 
   def team
+    @title = "The Exercism Team"
```

Comment:
I think we should name this variable `@page_title` or `@meta_title` or something, to avoid risking clashes. Thoughts?

Reply:
Also, what's your thinking about doing this here rather than in the helper like the other ones?

Reply:
I decided to put this here since all actions which goes through the page controller uses the `@title` variable.  I agree with changing the variable name to `@page_title`.